logic/manager/VersionManager.cs

```diff
@@ -34,17 +34,27 @@ private VersionManager()
                     Application.Current?.Dispatcher?.InvokeAsync(() => vanillaVersions.Add(version),
                         DispatcherPriority.Background);
                 }
+                versions = await WebRequestManager.Instance.GetSnapshotVersions();
```

Comment:
Here you can use the vanilla access with another version type (see comment above)